tests/acceptance/features/apiGraph/unassignRole.feature

```diff
@@ -0,0 +1,96 @@
+@api
+Feature: delete role
+  As an admin
+  I want to delete roles of users
+  So that the role of user is set to default
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+  @issue-5938
+  Scenario: admin user unassign the role of another user with admin role using graph API
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And the administrator has assigned the role "Admin" to user "Brian" using the Graph API
+    When the user "Alice" unassigns the role of user "Brian" using the Graph API
+    Then the HTTP status code should be "204"
+    And user "Brian" should not have any role assigned
+    # issue-5938 is related to below step, remove the comment when the issue is fixed
+    And user "Brian" should not be able to create a group "mygroup"
+    And user "Brian" should have the role "User" assigned
```

Comment:
`And user "Brian" should not have any role assigned` followed by `And user "Brian" should have the role "User" assigned` seems a little bit contradictory to me. Can we say this sentence `And user "Brian" should not have any role assigned` in some specific way so it doesn't contradict the below then step. what kind of role is this? App role? 

```suggestion
    And user "Brian" should not have any app role assigned
    # issue-5938 is related to below step, remove the comment when the issue is fixed
    And user "Brian" should not be able to create a group "mygroup"
    And user "Brian" should have the role "User" assigned
```

Reply:
I guess app roles stand for all user roles (i.e. admin, space, guest, user) and user role also points to the same thing. But i am not sure

Reply:
true @amrita-shrestha issue tag should be enough we decided not to add redundant comments